server/sonar-web/src/main/js/apps/account/organizations/CreateOrganizationForm.js

```diff
@@ -229,7 +229,7 @@ class CreateOrganizationForm extends React.PureComponent {
           </div>
 
           <footer className="modal-foot">
-            {this.state.processing
+            {this.state.loading
```

Comment:
I think it was done this way to prevent the use to use the cancel button ? Since you can't really cancel the current operation...
So should I display the spinner and disable both buttons ?